Makefile

```
@@ -368,3 +368,9 @@ else
 		-r $(REPO_OWNER)/$(REPO_NAME) \
 		-Z > codecov-upload.log
 endif
+
+NEW-VERSION ?= "0.0.23"
+.PHONY: update-version
+update-version:
+	chmod +x ./hack/update-version.sh
+	./hack/update-version.sh $(NEW-VERSION)
```

IMHO: Instead of new variable `NEW-VERSION` we might be able to use an existing variable of version `OPERATOR_VERSION`. It will reduce your work of updating one more variable. Can you once try this way?
```suggestion
	./hack/update-version.sh $(OPERATOR_VERSION)
```
And if the operator version is supposed to update by `+1` i.e `n->n+1` then we can directly embed this logic in code so we won't need to pass any command-line args. WDYT?

---

Removing Makefile target because scripts under hack/ can be run directly. and it is more convenient to do the same by directly passing arguments rather than passing via env variables through Makefile.